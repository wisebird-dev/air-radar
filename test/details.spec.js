import React from 'react';
import {shallow} from 'enzyme';

import Details from '../src/components/details';

test('it works', () => {
	const wrapper = shallow(<Details/>);
	expect(wrapper).toMatchSnapshot();
});
