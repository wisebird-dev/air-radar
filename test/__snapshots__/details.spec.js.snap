// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    margin: 1em 0;

    summary {
      cursor: pointer;
  }
    summary::-webkit-details-marker {
      font-size: 0.8em;
      margin-right: 0.8em
  }
",
        ],
      },
      "displayName": "styled.details",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "details",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
